engine/src/main/resources/assets/i18n/menu_fr.lang

```diff
@@ -348,13 +348,13 @@
     "start-singleplayer-game": "Solo",
     "start-playing": "Commencer à jouer",
     "storage-service": "Service de stockage d'identité",
-    "storage-service-conflict-message": "Lors de la dernière synchronisation des identités de client stockées localement avec le service de stockage, un conflit a été détecté. Pour le serveur avec l'ID %s, le certificat stocké localement a l'ID %s, tandis que le certificat distant a l'ID %s. Qu'est-ce que tu veux faire? Veuillez noter que garder l’une des versions supprimera l’autre; ignorer conservera l'identité locale localement et l'identité distante à distance, et le conflit sera détecté à nouveau lors de la prochaine synchronisation",
+    "storage-service-conflict-message": "Lors de la dernière synchronisation des identités de client stockées localement avec le service de stockage, un conflit a été détecté. Pour le serveur avec l'ID %s, le certificat stocké localement a l'ID %s, tandis que le certificat distant a l'ID %s. Qu'est-ce que tu veux faire ? Veuillez noter que garder l’une des versions supprimera l’autre; ignorer conservera l'identité locale localement et l'identité distante à distance, et le conflit sera détecté à nouveau lors de la prochaine synchronisation",
     "storage-service-conflict-keep-local": "Conserver la version locale (écraser sur le service)",
     "storage-service-conflict-keep-remote": "Conserver la version distante (écraser en local)",
     "storage-service-conflict-ignore": "Ignorer",
     "storage-service-log-in": "S'identifier",
     "storage-service-log-out": "Déconnexion",
-    "storage-service-log-out-popup": "Vous êtes sur le point de déconnecter votre compte de ce client. Voulez-vous supprimer les identités de client stockées localement? Si vous ne les supprimez pas, ils seront téléchargés sur le compte du prochain utilisateur qui se connecte. Choisissez Oui si vous souhaitez vous connecter avec un autre compte; choisissez Non si vous souhaitez vous reconnecter à ce compte après votre déconnexion. Choisissez Annuler si vous ne voulez pas vous déconnecter du tout.",
+    "storage-service-log-out-popup": "Vous êtes sur le point de déconnecter votre compte de ce client. Voulez-vous supprimer les identités de client stockées localement ? Si vous ne les supprimez pas, ils seront téléchargés sur le compte du prochain utilisateur qui se connecte. Choisissez Oui si vous souhaitez vous connecter avec un autre compte; choisissez Non si vous souhaitez vous reconnecter à ce compte après votre déconnexion. Choisissez Annuler si vous ne voulez pas vous déconnecter du tout.",
```

Comment:
I'm not a french speaker apart from some tiny bits of knowledge, but based on my quick google search, shouldn't there be a spece before the `;`?
```suggestion
    "storage-service-log-out-popup": "Vous êtes sur le point de déconnecter votre compte de ce client. Voulez-vous supprimer les identités de client stockées localement ? Si vous ne les supprimez pas, ils seront téléchargés sur le compte du prochain utilisateur qui se connecte. Choisissez Oui si vous souhaitez vous connecter avec un autre compte ; choisissez Non si vous souhaitez vous reconnecter à ce compte après votre déconnexion. Choisissez Annuler si vous ne voulez pas vous déconnecter du tout.",
```